src/components/Login/LoginMobileNumberCard.test.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  render,
+  fireEvent,
+  cleanup,
+  waitFor,
+} from "@testing-library/react-native";
+import React from "react";
+import { Sentry } from "../../utils/errorTracking";
+import { AlertModalContextProvider } from "../../context/alert";
+import { CreateProvidersWrapper } from "../../test/helpers/providers";
+import { LoginMobileNumberCard } from "./LoginMobileNumberCard";
+import "../../common/i18n/i18nMock";
+
+jest.mock("../../utils/errorTracking");
+const mockCaptureException = jest.fn();
+(Sentry.captureException as jest.Mock).mockImplementation(mockCaptureException);
+
+const setLoginState = jest.fn();
+const setMobileNumber = jest.fn();
+const setCountryCode = jest.fn();
+const mockHandleRequestOTP = jest.fn().mockResolvedValue(true);
+
+describe("LoginMobileNumberCard", () => {
+  afterEach(() => {
+    cleanup();
+    jest.resetAllMocks();
+  });
+
+  it("should be able to fetch the endpoint without error", async () => {
```

Comment:
This test name is not very descriptive, perhaps you can express what the test is trying to assert.
e.g. "should request for OTP successfully on pressing submit"